src/app/modules/item/pages/item-details/item-details.component.ts

```diff
@@ -52,7 +52,7 @@ export class ItemDetailsComponent implements OnDestroy, BeforeUnloadComponent {
   );
 
   readonly formerAnswerError$ = this.formerAnswer$.pipe(
-    catchError(error => of(errorIsHTTPForbidden(error) ? loadForbiddenAnswerError : new Error(error))),
+    catchError((error: string) => of(errorIsHTTPForbidden(error) ? loadForbiddenAnswerError : new Error(error))),
```

Comment:
How do you know for sure it is a string, I don't think you can do this assumption.

Reply:
fixed